pkg/controllers/updaterun/initialization.go

```diff
@@ -404,7 +407,7 @@ func validateAfterStageTask(tasks []placementv1beta1.AfterStageTask) error {
 	}
 	for i, task := range tasks {
 		if task.Type == placementv1beta1.AfterStageTaskTypeTimedWait {
-			if task.WaitTime.Duration <= 0 {
+			if task.WaitTime != nil && task.WaitTime.Duration <= 0 {
```

Comment:
Should we use "if task.WaitTime == nil || task.WaitTime.Duration <= 0"? We require users to provide a value in this case. There could be chance that the code is released but the CRD with CEL check has not been updated to latest.